utils/generateHTML.js

```diff
@@ -1,58 +1,77 @@
 const { readFile, writeFile } = require("fs/promises");
 const path = require("path");
 const createHtml = require("create-html");
+var htmlBody = "";
```

Comment:
You should change this line into `let`, as using `var` might cause some unexpected behaviour